src/Stratis.CirrusD/Program.cs

```diff
@@ -60,10 +64,30 @@ private static IFullNode GetSideChainFullNode(NodeSettings nodeSettings)
                 .UseSmartContractWallet()
                 .UseApi()
                 .AddRPC()
-                .UseDiagnosticFeature()
-                .Build();
+                .UseDiagnosticFeature();
 
-            return node;
+            if (nodeSettings.EnableSignalR)
+            {
+                nodeBuilder.AddSignalR(options =>
```

Comment:
@ianadavies it might be a good idea to add this piece of code to an extension method or something in the SignalR feature? So that we don't duplicate like this 👍

Reply:
I think letting node users to decide what events to listen to is better, so the way it is now is good. We can maybe have a default set of events and put it into a reusable extension, but not sure if it adds any value.

Reply:
@dangershony we are going to make these changes in a following PR 👍